### ISSUE TITLE
[Editorial] platform authenticator relationship to WebAuthn Client and Client Device

### ISSUE THREAD

**Issue by timcappalli** [w3c/webauthn#2164]
https://w3c.github.io/webauthn/#webauthn-client-device

> [platform authenticators](https://w3c.github.io/webauthn/#platform-authenticators) are bound to a [client device](https://w3c.github.io/webauthn/#client-device) rather than a [WebAuthn Client](https://w3c.github.io/webauthn/#webauthn-client).

This isn't always true. Update text.

**Comment by emlun** [comment]
No, I argue this is in fact always true, but that the _client device_ can also act as a [roaming authenticator](https://w3c.github.io/webauthn/#roaming-authenticators) in some contexts. Whether a given authenticator is a _platform authenticator_ or a _roaming authenticator_ is decided by the client executing a WebAuthn ceremony, not by intrinsic properties of the authenticator itself. An Android phone "is" a platform authenticator when executing WebAuthn in a browser running on the phone, but "is" a roaming authenticator when acting as a Bluetooth authenticator with a client running on a laptop.

If those are unhelpful definitions, then we would instead need to replace the definitions with new ones.

**Comment by emlun** [comment]
That example is described in [§6.2.1. Authenticator Attachment Modality](https://w3c.github.io/webauthn/#sctn-authenticator-attachment-modality):

>Some [platform authenticators](https://w3c.github.io/webauthn/#platform-authenticators) could possibly also act as [roaming authenticators](https://w3c.github.io/webauthn/#roaming-authenticators) depending on context. For example, a [platform authenticator](https://w3c.github.io/webauthn/#platform-authenticators) integrated into a mobile device could make itself available as a [roaming authenticator](https://w3c.github.io/webauthn/#roaming-authenticators) via Bluetooth. In this case [clients](https://w3c.github.io/webauthn/#client) running on the mobile device would recognise the authenticator as a [platform authenticator](https://w3c.github.io/webauthn/#platform-authenticators), while [clients](https://w3c.github.io/webauthn/#client) running on a different [client device](https://w3c.github.io/webauthn/#client-device) and communicating with the same authenticator via Bluetooth would recognize it as a [roaming authenticator](https://w3c.github.io/webauthn/#roaming-authenticators).

**Comment by timcappalli** [comment]
There are cases where the authenticator is bound only to the WebAuthn client, and not the underlying client device. 

Examples:

1. Google Password Manager in Chrome
2. A passkey provider operating as a browser extension (which is itself the WebAuthn client)

**Comment by emlun** [comment]
Ok, those are fair counterexamples.
